mlflow/gateway/constants.py

```diff
@@ -0,0 +1,6 @@
+GATEWAY_VERSION = "0.1.0"
+CONF_PATH_ENV_VAR = "MLFLOW_GATEWAY_CONFIG_PATH_ENV_VAR_NAME"
+LOCALHOST = "127.0.0.1"
+SERVER_HOUSEKEEPING_PAUSE = 2
+
+PROVIDERS = {"openai", "anthropic", "databricks"}
```

Comment:
We're also going to want to introduce an MLflow Served Models provided

```suggestion
PROVIDERS = {"openai", "anthropic", "databricks_serving_endpoint", "mlflow_served_model"}
```

Reply:
updated with configs and Provider class entries